README.md

```diff
@@ -58,19 +52,25 @@ To use the services provided by this clients, they need to be instantiated in sp
    }
 ``` 
 
+There is a factory that creates a default implementation which provides JWT auto refreshing and bearer wrapping features:
```

Comment:
Maybe that could be the 'default' way? Ie use the factory in bean configuration example above.